rock/orogen_plugin.rb

```diff
@@ -116,129 +116,127 @@ def self.add_to(task)
         end
         task.register_extension Extension.new('port_listener', task)
     end
+
+    def self.resolve_time_field(type_name, time_field, sample_name)
+        is_pointer = type_name.include? "::RTT::extras::ReadOnlyPointer"
```

Comment:
In transformer the check was `stream_data_type =~ /ReadOnlyPointer/` which is more "open". Is it ok?

Reply:
I think so. This is the full name.